Library/Formula/logentries.rb

```diff
@@ -20,4 +21,29 @@ def install
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+        <string>#{opt_bin}/le</string>
+        <string>monitor</string>
+        </array>
+        <key>KeepAlive</key>
+        <true/>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    system "if [ \"`#{bin}/le --help || echo $?`\" -ne \"4\" ]; then exit 1; fi"
```

Comment:
@mikemcquaid the problem is that any sort of configuration will expose some private token to the world which isn't good from a security point of view (even if it is a throw away account). Any service which requires registration before it becomes active would have this problem. The other alternative is we can check to see if a more specific command works but we will still be doing a test expecting to see an error message.